website/versioned_docs/version-v2.0_alpha/developers-references/smart-contracts/Params.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Params Smart Contract
+description: Params smart contract
+sidebar_label: Params
+sidebar_position: 9
+---
+
+A contract holding three structs:
+
+```c
```

Comment:
Is it an error? Maybe it should be solidity or javascript?

Reply:
couldn't get highlighting with solidity so to highlight the struct had to use c or go (js wouldn't highlight properly)